src/speed_trap_options.erl

```diff
@@ -80,3 +92,20 @@ validate_override(blocked) ->
   ok;
 validate_override(Override) ->
   {error, {override, Override}}.
+
+validate_scaling_time_interval(TimeInterval) when is_integer(TimeInterval), TimeInterval > 0 ->
+  ok;
+validate_scaling_time_interval(TimeInterval) ->
+  {error, {invalid_time_interval, TimeInterval}}.
+
+validate_rejection_rate_threshold(Threshold)
+  when is_number(Threshold), Threshold >= 0, Threshold =< 100 ->
```

Comment:
Shouldn't this be `Threshold > 0`? 

README states:
> rejection_rate_threshold => 30, % Upscale if >30% rejected

How will this behave if rejection_rate_threshold is set to 0? Then there's effectively no upscaling which feels kinda strange? 😅

Reply:
It should be `> 0`. Will fix it